dragula.js

```diff
@@ -82,11 +84,30 @@ function dragula (initialContainers, options) {
     var offset = getOffset(_item);
     _offsetX = getCoord('pageX', e) - offset.left;
     _offsetY = getCoord('pageY', e) - offset.top;
-    renderMirrorImage();
-    drag(e);
+
+    renderMirrorAndDrag(e);
+
     e.preventDefault();
   }
 
+  function renderMirrorAndDrag(e){
+
+    if(o.delay > 0){
+
+      _timerClick = setTimeout(function(){
```

Comment:
As mentioned earlier, please extract this and just call that function. E.g

```
if (o.delay) {
  _timerClick = setTimeout(foo)
} else {
  foo();
}
```

This way you avoid the else, code repetition, and ciclomatic complexity